Divide_and_Conquer/maximum-subarray.py

```diff
@@ -0,0 +1,63 @@
+# Problem
+
+# Task:
+# You are given a one dimensional array that may contain both positive and negative integers.
+# find the sum of contiguous subarray of numbers which has the largest sum.
+
+# Input Format. The input consists of a one dimensional space separated array.
+
+# Constraints. 1 ≤ m ≤ 10**3 .
+
+# Output Format. Output the starting index, ending index and value of sum of subarray.
+
+# Example :
+# -3 -8 6 -2 -3 1 5 -6                       <--- input
+#  2 6 7                                     <--- output
+
+import math as m
```

Comment:
I checked this file https://ide.codingblocks.com/s/141194.
It shows runtime error. Can you please check:)

Reply:
The error is coming due to space after the input. I have now fixed this bug.
If it is still giving error, make sure to change the language from Python to Python3.
You can check out here too https://ide.geeksforgeeks.org/I8pm7SjKF0
Thanks :)

Reply:
Yeah Thanks for the changes